### PR TITLE
fix isolated local_setup.bat

### PR DESCRIPTION
**Pull request by dirk-thomas**
Addresses ros2/ros2#17

### CODE REVIEW

File: ament_package/template/isolated_prefix_level/local_setup.bat.in [hunk 1]
```diff
@@ -1,53 +1,43 @@
 :: generated from ament_package/template/isolated_prefix_level/local_setup.sh.in
```
Comment (wjwwood) [inline ament_package/template/isolated_prefix_level/local_setup.bat.in]: This is the wrong source file.

File: ament_package/template/isolated_prefix_level/local_setup.bat.in [hunk 2]
```diff
@@ -1,53 +1,43 @@
...
+    set "ordered_packages=!ordered_packages!%%p"
+  )
+  endlocal & (
+    set "%~1=%ordered_packages%"
   )
 goto:eof
+
+:call_file
+  if "%AMENT_TRACE_SETUP_FILES%" NEQ "" echo call "%~1"
+  if exist "%~1" call "%~1%"
```
Comment (wjwwood) [inline ament_package/template/isolated_prefix_level/local_setup.bat.in]: Should the trace message be printed if the file doesn't exist?

Comment (dirk-thomas) [inline ament_package/template/isolated_prefix_level/local_setup.bat.in]: Non of the setup scripts currently prints when files are missing. I kept it the same way in this PR.

Comment (wjwwood) [inline ament_package/template/isolated_prefix_level/local_setup.bat.in]: Ok.

**Comment by wjwwood** [comment]
lgtm

File: ament_package/template/isolated_prefix_level/local_setup.bat.in (hunk 1 above)
Comment (dirk-thomas) [inline ament_package/template/isolated_prefix_level/local_setup.bat.in]: Fixed in 1539bed35d5465e91afc7bb13313972d5d97d6a0

**Pull request merged**